build.gradle

```diff
@@ -92,6 +92,11 @@ allprojects {
     tasks.withType(DockerBuildImage).all { task ->
         task.outputs.upToDateWhen { false }
     }
+
```

Comment:
Shoudln't that be in the subprojects (`// Java projects common configurations`) part?

Reply:
oops yes